scripts/lib/CIME/SystemTests/system_tests_compare_two.py

```diff
@@ -161,28 +193,38 @@ def run_phase(self, success_change=False):  # pylint: disable=arguments-differ
         Runs both phases of the two-phase test and compares their results
         If success_change is True, success requires some files to be different
         """
+        first_phase = self._case1.get_value("RESUBMIT") == 1
```

Comment:
This first_phase variable was confusing when I first read it. Maybe add a comment like, "Only relevant when self._multisubmit is True"?